### ISSUE TITLE
secDNS causing trouble on CentralNIC EPP

### ISSUE THREAD

**Issue by tmuncks** [InternetNZ/pyepp#55]
**Describe the bug**
In some situations, getting domain info for a DNSSEC enabled domain in CentralNIC causes problems:

```
Traceback (most recent call last):
  File "/home/thms/projects/epp/test.py", line 35, in <module>
    domain_check = domain.info('solutions.gl')
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/thms/projects/epp/venv/lib64/python3.11/site-packages/pyepp/domain.py", line 182, in info
    'flag': raw_response.find('flag').text,
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

The reason appears to be CentralNIC using unexpected names under the `dns_key` sections. 

If I change the names in both `dns_key` sections in `domain.py` function definition for `info()`, changing:

`flag` -> `flags`
`algorithm` -> `alg`
`public_key` -> `pubKey`

Like this:
```
'dns_key': {
        #'flag': raw_response.find('flag').text,
        'flag': raw_response.find('flags').text,
        'protocol': raw_response.find('protocol').text,
        #algorithm': raw_response.find('algorithm').text,
        'algorithm': raw_response.find('alg').text,
        #'public_key': raw_response.find('public_key').text,
        'public_key': raw_response.find('pubKey').text,
}
```
...it starts working for me. This is obviously not a solution, as it would most likely break something else.

**To Reproduce**
This is a real life raw_response from CentralNIC, of a response that causes the problem.

```
<?xml version="1.0" encoding="utf-8"?>
<epp xmlns="urn:ietf:params:xml:ns:epp-1.0">
    <response>
        <result code="1000">
            <msg>Command completed successfully.</msg>
        </result>
        <resData>
            <domain:infData xmlns:domain="urn:ietf:params:xml:ns:domain-1.0">
                <domain:name>solutions.gl</domain:name>
                <domain:roid>D400779210-CNIC</domain:roid>
                <domain:status s="serverTransferProhibited"/>
                <domain:status s="clientTransferProhibited"/>
                <domain:status s="clientDeleteProhibited"/>
                <domain:registrant>REDACTED</domain:registrant>
                <domain:contact type="tech">REDACTED</domain:contact>
                <domain:contact type="admin">REDACTED</domain:contact>
                <domain:contact type="billing">REDACTED</domain:contact>
                <domain:ns>
                    <domain:hostObj>ns1.modulo.gl</domain:hostObj>
                    <domain:hostObj>ns2.modulo.gl</domain:hostObj>
                    <domain:hostObj>ns3.modulo.gl</domain:hostObj>
                </domain:ns>
                <domain:clID>REDACTED</domain:clID>
                <domain:crID>REDACTED</domain:crID>
                <domain:crDate>2023-10-02T17:05:17.0Z</domain:crDate>
                <domain:upDate>2023-10-02T17:46:31.0Z</domain:upDate>
                <domain:exDate>2024-10-02T23:59:59.0Z</domain:exDate>
            </domain:infData>
        </resData>
        <extension>
            <secDNS:infData xmlns:secDNS="urn:ietf:params:xml:ns:secDNS-1.1">
                <secDNS:dsData>
                    <secDNS:keyTag>17600</secDNS:keyTag>
                    <secDNS:alg>13</secDNS:alg>
                    <secDNS:digestType>1</secDNS:digestType>
                    <secDNS:digest>4ffb0f3f0fb5edbaa0067d1cd94bb545249cfea2</secDNS:digest>
                    <secDNS:keyData>
                        <secDNS:flags>257</secDNS:flags>
                        <secDNS:protocol>3</secDNS:protocol>
                        <secDNS:alg>14</secDNS:alg>
                        <secDNS:pubKey>fHiagyGUIxsDmeYaaKOWceffQn7QTaCwM1y6Oy1zYRpndxu/6WAHM99p/X/dM7t+RcGahZ8+GlX5JHkbyaMllg==</secDNS:pubKey>
                    </secDNS:keyData>
                </secDNS:dsData>
                <secDNS:dsData>
                    <secDNS:keyTag>17600</secDNS:keyTag>
                    <secDNS:alg>13</secDNS:alg>
                    <secDNS:digestType>2</secDNS:digestType>
                    <secDNS:digest>024c96b094aaad3892babb303e02587f7d6f8f749fef9ff642e3bb8bfc2e0dcf</secDNS:digest>
                    <secDNS:keyData>
                        <secDNS:flags>257</secDNS:flags>
                        <secDNS:protocol>3</secDNS:protocol>
                        <secDNS:alg>13</secDNS:alg>
                        <secDNS:pubKey>fHiagyGUIxsDmeYaaKOWceffQn7QTaCwM1y6Oy1zYRpndxu/6WAHM99p/X/dM7t+RcGahZ8+GlX5JHkbyaMllg==</secDNS:pubKey>
                    </secDNS:keyData>
                </secDNS:dsData>
                <secDNS:dsData>
                    <secDNS:keyTag>17600</secDNS:keyTag>
                    <secDNS:alg>13</secDNS:alg>
                    <secDNS:digestType>4</secDNS:digestType>
                    <secDNS:digest>14d92f9f1e82c0906e2003b9e3401de4024f705724ef14d848fc2708d5714d5572db040faa3e317753e6577c7e70cbb8</secDNS:digest>
                    <secDNS:keyData>
                        <secDNS:flags>257</secDNS:flags>
                        <secDNS:protocol>3</secDNS:protocol>
                        <secDNS:alg>13</secDNS:alg>
                        <secDNS:pubKey>fHiagyGUIxsDmeYaaKOWceffQn7QTaCwM1y6Oy1zYRpndxu/6WAHM99p/X/dM7t+RcGahZ8+GlX5JHkbyaMllg==</secDNS:pubKey>
                    </secDNS:keyData>
                </secDNS:dsData>
            </secDNS:infData>
            <rgp:infData xmlns:rgp="urn:ietf:params:xml:ns:rgp-1.0">
                <rgp:rgpStatus s="addPeriod"/>
            </rgp:infData>
            <regtype:infData xmlns:regtype="urn:ietf:params:xml:ns:regtype-0.1">
                <regtype:type>standard</regtype:type>
            </regtype:infData>
        </extension>
        <trID>
            <clTRID>ff3e31b0-ae6d-423a-8f3f-357450fcfc13</clTRID>
            <svTRID>CNIC-4DE6209B97F6572B4C7A90B13ED17B3F0235778891E0043EA15BB78111A</svTRID>
        </trID>
    </response>
</epp>
```

**Expected behaviour**
Everything should work and return the info present in the domain

**Desktop (please complete the following information):**
 - OS: Fedora Linux
 - Version 38

**Python version**
 - Python 3.11.5

**Additional context**
Running out of a venv with nothing but pyepp installed.

I have

**Comment by ehsan-fj** [comment]
Thanks for submitting the issue. That's definitely a typo in the code and a bug. It fill be fixed soon.

**Comment by tmuncks** [comment]
Cool, thanks

**Comment by ehsan-fj** [comment]
Fixed in version [0.0.5](https://pypi.org/project/pyepp/0.0.5/)

**Comment by tmuncks** [comment]
Awesome - Thank you!
